console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/client/schedule/JobScheduleAddDialog.java

```diff
@@ -87,9 +94,11 @@ public void createBody() {
 
         endsOn.setFieldLabel(JOB_MSGS.dialogAddScheduleEndsOnLabel());
         endsOn.setFormatValue(true);
+        endsOn.setAllowBlank(false);
         endsOn.getPropertyEditor().setFormat(DateTimeFormat.getFormat("dd/MM/yyyy"));
         mainPanel.add(endsOn);
 
+        endsOnTime.setAllowBlank(false);
```

Comment:
Why is this required? A schedule is not required to have an ending, meaning that it will be valid forever. Is this case covered somewhere else?